resources/js/components/ExpRightNav.vue

```diff
@@ -3,17 +3,20 @@
 		<div class="containerR" id="tools" style="height:100%;">	
 			
 		   	<div class="input-alt"></div>	
-		   	<span v-if="vicelab==1" class="pr z-1" >		 
+		   	<span v-if="vicelab==true" class="pr z-1" >		 
```

Comment:
You too like vicelab this guy,  why not use a descriptive variable name instead? Please correct. @saidabdul80

Reply:
please correct accordingly